core/controllers/editor.py

```diff
@@ -288,12 +288,15 @@ def _get_exploration_data(
                 exploration_id, exp_user_data.draft_change_list_exp_version)
             if exp_user_data and exp_user_data.draft_change_list_exp_version
             else None)
+        draft_id = (exp_user_data.draft_change_list_id if exp_user_data
```

Comment:
Can we just call this draft_change_list_id throughout, for consistency? I know it's longer but then at least we won't have both a draft_id and a draft_change_list_id.